packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/customize-link-in-widget/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Customize the sign-up link in the Widget
+---
+
+If you left the <GuideLink link="../configure-self-service-registration-policy">**Add to Sign-In widget**</GuideLink> check box clear when configuring the registration policy, then you need to configure a link using JavaScript in the **Customize Sign-In Page** HTML editor.
+
+> **Note:** To enable the Custom Sign-In Page HTML editor, you must have a [custom URL domain](/docs/guides/custom-url-domain/overview/) configured.
+
+1. In the Admin Console, go to **Settings** and then **Customization**.
+2. Click the **Custom Sign In** tab.
+3. In the HTML editor, add the following configuration parameters directly under the `var config = OktaUtil.getSignInWidgetConfig();` line:
+
+```JavaScript
+config['features.registration'] = true;
+config['authScheme'] = 'SESSION';
+```
+
+4. Click **Save and Publish**.
```

Comment:
Add some sort of "next" button or list of guides after this

Reply:
This is going to be handled by @albertchen-okta and @alexdahl-okta ?

Reply:
I think each guide author should add whatever next steps are appropriate. The authors will know better than we will!
I think it's "just markdown": a `Next steps` page and list of suggested topics and resources.

We have a bunch of examples in guides today, such as https://developer.okta.com/docs/guides/sign-into-spa/angular/next-steps/
But any improvements to format or tailoring to specific guides is very welcome!